<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConfigDialog</class>
 <widget class="QDialog" name="ServerConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Config Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="hostLineEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;jijijij&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="hostErrorMsg">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="portLineEdit">
       <property name="whatsThis">
        <string>fwfwfww</string>
       </property>
       <property name="placeholderText">
        <string>6379</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="portErrorMsg">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>If not,leave it blank</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="passwordErrorMsg">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServerConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServerConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
